src/wallet/wallet.cpp

```diff
@@ -1594,14 +1594,15 @@ bool CWallet::ImportScriptPubKeys(const std::string& label, const std::set<CScri
     return true;
 }
 
-int64_t CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *wallet, bool use_max_sig)
+// Returns pair of vsize and weight
+std::pair<int64_t, int64_t> CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *wallet, bool use_max_sig)
```

Comment:
I have noticed that this codebase often uses unnamed fields in a specific order rather than named fields. Wouldn't it be advantageous to define transfer objects that have named fields to be more explicit in what is retrieved from response objects?

Reply:
For something like this, I feel like it is overkill to introduce a struct or class for these, especially since it is used in only one place.